bitcoin/src/crypto/key.rs

```diff
@@ -230,13 +227,53 @@ impl fmt::Display for PublicKey {
     }
 }
 
+/// Error returned while constructing public key from string
+#[derive(Debug, PartialEq)]
+pub enum ParsePubKeyError {
```

Comment:
In 4717cb9a384afc21817a1fba35573311e4ce2b0b:

Would prefer this be called `ParsePublicKeyError`. I know it's long and annoying, but users should basically never need to type this, and if they *do* need to type it, it should be minimally surprising.

Though come to think of it, could also probably be called `ParseError` since it's already in the `key` module.

Reply:
Ah, can't be called `ParseError` since it needs to distinguish from similar `CompressedPublicKeyError`.